frontend/packages/ux-editor/src/components/Properties/PropertiesHeader/EditLayoutSetForSubform/EditLayoutSet/CreateNewLayoutSet/index.ts

```diff
@@ -0,0 +1 @@
+export { CreateNewLayoutSet } from './CreateNewLayoutSet';
```

Comment:
If this is specific to Subform, maybe we should indicate this in the naming?

Reply:
I changed the name to `CreateNewSubformLayoutSet`

Reply:
Nice!